### DISCUSSION TITLE
LAB #A Ch6 Analysis and Optimization - Creating a Vitis HLS Project - Board selection

### DISCUSSION THREAD

**Discussion by JasonYanggg** [bol-edu/HLS-SOC-Discussions#38] [Q&A] [answered]
Hi, I have been doing Ch6 in Lab A. In the tutorial, we are asked to select U200 when creating the project. However, I can not find it on the server and my computer. Therefore, I tried to search for possible solutions on the Internet.

I tried to modify the board_supported.csv and part_supported.csv, but it doesn't work (https://support.xilinx.com/s/question/0D52E00006txp5wSAA/mode-composer-cannot-find-u200-board?language=zh_CN). 

My computer runs Windows, so I can't install the package of U200. 

In my opinion, the only method to run the tutorial would be to choose another boards rather than U200 (https://support.xilinx.com/s/question/0D54U00005kKv7NSAS/how-to-fix-error-hls-2001023-part-is-not-installed-?language=en_US). However, I have no idea if this method is allowed in the homework.

Is there any suggestion? Thank you.

**Comment by kevinjantw** [comment]
Are you NTHU student? pls contact your HLS course TA.
The boled lab servers don't support Xilinx U200 board and its tutorial currently.

**Reply by JasonYanggg** [reply]
Yes, I am. I will contact them. Thank you!

**Comment by jiinlai** [comment, marked as answer]
please choose the board U50 as in Lab3.

Jiin

楊博舜 ***@***.***>於 2023年3月3日 週五，下午9:48寫道：
